kiliautoml/models/_hugging_face_text_classification_model.py

```diff
@@ -78,6 +78,11 @@ def train(
 
         path_dataset = os.path.join(PathHF.dataset_dir(model_repository_dir), "data.json")
         logger.info(f"Downloading data to {path_dataset}")
+        if not clear_dataset_cache:
```

Comment:
why only for text classification? that's probably the least used task.

Reply:
cache is not implemented in NER, I created a branch to implement it for NER but I think we should make another ticket for that

Reply:
👍